usecases/ast_eval/evaluate/eval_string_starts_with.go

```diff
@@ -0,0 +1,44 @@
+package evaluate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/errors"
+
+	"github.com/checkmarble/marble-backend/models/ast"
+)
+
+type StringStartsWith struct {
```

Comment:
As discussed orally: let's put the two in just one class, receiving the function as input